src/client.js

```diff
@@ -572,6 +572,27 @@ MatrixClient.prototype.rehydrateDevice = async function() {
     }
 };
 
+/**
+ * Get the current dehydrated device, if any
+ * @return {Promise} A promise of an object containing the dehydrated device
+ */
+MatrixClient.prototype.getDehydratedDevice = async function() {
```

Comment:
The boundary between base APIs and client is quite blurry, but perhaps this one is more base APIs, since it's just a straightforward API access...?

Reply:
hm, i think it's probably the lesser evil to leave it next to the rest of the dehydration functions, even though it's just an API wrapper?  and it makes the code easier to read...

Reply:
Okay, seems alright then. As mentioned, it's quite a blurry line anyway.